src/authentication.ts

```diff
@@ -80,9 +80,10 @@ function isTokenResponse(body: unknown | TTokenResponse): body is TTokenResponse
 function postTokenRequest(
   tokenEndpoint: string,
   tokenRequest: TTokenRequest,
-  credentials: RequestCredentials
+  credentials: RequestCredentials,
+  headers?: TPrimitiveRecord
 ): Promise<TTokenResponse> {
-  return postWithXForm({ url: tokenEndpoint, request: tokenRequest, credentials: credentials }).then((response) => {
+  return postWithXForm({ url: tokenEndpoint, request: tokenRequest, credentials: credentials, headers }).then((response) => {
```

Comment:
If you look at `credentials` in the same line, we've expanded it to say `credentials: credentials`.

Now, I'm all for using the short-form, but if we're gonna use it, the line should at least be consistent about it. Could you update `credentials`, or expand to `headers: headers`? Either is fine.